### PR TITLE
Add the 7.0.17 release to the website

### PR DESCRIPTION
**Pull request by OndroMih**

### CODE REVIEW

**Review by avpinchuk — APPROVED** [review]

File: docs/website/src/main/resources/download.md [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
+In this summer 2024 release of GlassFish we did a lot of maintenance in which we will continue in future releases. We fixed several security issues and the build now passes also in Java 23. We welcome feedback - if you notice that something doesn't work as expected, feel free to create an issue or even better create your own pull request.
 
 Download:
 
-* [Eclipse GlassFish 7.0.16, Jakarta EE Platform, 10](https://www.eclipse.org/downloads/download.php?file=/ee4j/glassfish/glassfish-7.0.16.zip)
-* [Eclipse GlassFish 7.0.16, Jakarta EE Web Profile, 10](https://www.eclipse.org/downloads/download.php?file=/ee4j/glassfish/web-7.0.16.zip)
-* [Eclipse GlassFish Embedded 7.0.16, Jakarta EE Platform, 10](https://search.maven.org/artifact/org.glassfish.main.extras/glassfish-embedded-all/7.0.16/jar)
-* [Eclipse GlassFish Embedded 7.0.16, Jakarta EE Web Profile, 10](https://search.maven.org/artifact/org.glassfish.main.extras/glassfish-embedded-web/7.0.16/jar)
+* [Eclipse GlassFish 7.0.17, Jakarta EE Platform, 10](https://www.eclipse.org/downloads/download.php?file=/ee4j/glassfish/glassfish-7.0.17.zip)
```
Comment (pzygielo) [inline docs/website/src/main/resources/download.md:13]: image

Comment (OndroMih) [reply docs/website/src/main/resources/download.md:13]: The file is really not there. Previous versions are there. @arjantijms do you know why?

Comment (arjantijms) [reply docs/website/src/main/resources/download.md:13]: Yes, the script https://ci.eclipse.org/glassfish/job/glassfish-website-publish-zips-from-staging/ wasn't executed. This should be done as part of releasing GlassFish. I'll run it now.

Comment (arjantijms) [reply docs/website/src/main/resources/download.md:13]: It's there now: https://download.eclipse.org/ee4j/glassfish/

**Pull request merged**